### ISSUE TITLE
Double clicking on summary row is causing crash

### ISSUE THREAD

**Issue by FlameHorizon** [moneymanagerex/moneymanagerex#5805]
MMEX version:
 - [ ] 1.6.0
 - [ ] 1.6.1
 - [ ] 1.6.2
 - [x] 1.6.3 Beta
 - [ ] Other (please specify)

Details version information:

![image](https://user-images.githubusercontent.com/23423558/230709708-8352ddf1-bfb4-46b0-9b1c-4b2770f6330d.png)

Note: bug reporters are expected to have verified the bug still exists
either in the last stable version of MMEX or on updated development code
(master branch).

Operating System:
 - [ ] Windows
 - [ ] Mac OSX
 - [x] Linux 

# Description of the bug

Double clicking on summary row in Budget Setup, causes application to crash.

# Reproduction

Is the bug reproducible? 
 - [x] Always 
 - [ ] Randomly 
 - [ ] Happened only once

Reproduction steps:

1. Create a new budget for year 2023
2. Open budget
3. Double click on blue row representing summary for category

Expected result:

Unknown.

Actual result:

Application crashes.

# Additional information

Small GIF representing error:

![Peek 2023-04-08 09-29](https://user-images.githubusercontent.com/23423558/230709572-174a76ca-32ad-499b-a812-31cb677bc2ad.gif)

**Comment by n-stein** [comment]
Duplicate of #5686, already fixed for 1.6.4.

**Comment by FlameHorizon** [comment]
I confirm. After updating to 1.6.4 beta.4 problem does not appear.
